starter/class/screen.js

```diff
@@ -91,10 +91,10 @@ class Screen {
     if (char.length !== 1) {
       throw new Error("invalid grid character");
     }
+    console.log(row, col, char)
```

Comment:
1. So starting from here, when you are finding out that it's undefined for some reason, you'll want to go back up the chain.  Where is `setGrid` being called from?